packages/block-library/src/image/image.js

```diff
@@ -964,16 +967,20 @@ export default function Image( {
 		<>
 			{ controls }
 			{ img }
-
-			<Caption
-				attributes={ attributes }
-				setAttributes={ setAttributes }
-				isSelected={ isSingleSelected }
-				insertBlocksAfter={ insertBlocksAfter }
-				label={ __( 'Image caption text' ) }
-				showToolbarButton={ isSingleSelected && hasNonContentControls }
-				readOnly={ lockCaption }
-			/>
+			{ /* Don't add caption component and controls in images with pattern overrides */ }
+			{ ! arePatternOverridesEnabled && (
+				<Caption
```

Comment:
Why do we need this? Can't we just disable the caption button?

Reply:
I just assumed we didn't need the `Caption` component. But I guess we could just disable the button. Maybe changing [this property](https://github.com/WordPress/gutenberg/pull/62747/files#diff-a14b61c874095649b236b78413e805b4541277b24eb71ce350a17d9f2b6396b1R978-R980)?